.github/workflows/swift-toolchain.yml

```diff
@@ -2205,7 +2325,8 @@ jobs:
               -p:PASSPHRASE=${{ secrets.PASSPHRASE }} `
               -p:DEVTOOLS_ROOT=${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain `
               -p:TOOLCHAIN_ROOT=${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain `
-              -p:SWIFT_INSPECT_BUILD=false `
+              -p:INCLUDE_SWIFT_INSPECT=${{ matrix.arch == 'amd64' && true || false }} `
+              -p:SWIFT_INSPECT_BUILD=${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr `
```

Comment:
Really? This is the path to the build?

Reply:
Updated to a directory outside of the directory we use for the staged tools.